README.md

```diff
@@ -1,28 +1,62 @@
 # MagicScript Components Lumin
 
-[![npm version](https://badge.fury.io/js/magic-script-components-lumin.svg)](https://badge.fury.io/js/magic-script-components-lumin) [![License](http://img.shields.io/:license-Apache%202.0-blue.svg)](LICENSE)
+[![npm version](https://badge.fury.io/js/magic-script-components-lumin.svg)](https://badge.fury.io/js/magic-script-components-lumin) [![License](https://img.shields.io/:license-Apache%202.0-blue.svg)](LICENSE)
 
-MagicScript Components renderer for Lumin Runtime platform
+MagicScript Component Framework for Lumin Runtime platform
+
+This package is meant to be used along with [MagicScript Components](https://github.com/magic-script/magic-script-components) and both should be included as dependencies in the project's `package.json`.
 
 ## Installation
 
+1. Install `magic-script-cli`:
+
+```bash
+npm install -g magic-script-cli
+```
+
+2. Create new MagicScript application:
+
+```bash
+magic-script init
+```
+
+- Follow the steps.
```

Comment:
Can we have it: Follow the steps and choose "Components" when application type is requested.